### PR TITLE
Abstract 1010CG Signature Content to Variables

### PR DESCRIPTION
**Pull request by LinnJS**
## Description
In efforts to make this content more digestible abstract content to variables so what content goes where is more trackable and less prone to errors

[ticket](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/18329)

## Testing done
- Visual testing

## Acceptance criteria
- [x] Static content is more maintainable and less prone to errors

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by LinnJS** [comment]
Would love feedback on this PR trying to create make content more maintainable and less prone to content going into wrong places due to the code being too confusing and out of order to content on the actual form.

**Review by craigwheeler — APPROVED** [review]

**Review by Mottie — APPROVED** [review]
ghe-emoji

**Review by timwright12 — APPROVED** [review]

**Comment by LinnJS** [comment]
@timwright12 thank you for the speedy unsolicited review!

File: src/applications/caregivers/tests/components/PreSubmitInfo/index.spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  secondaryCaregiverContent,
+} from 'applications/caregivers/definitions/content.js';
+
+describe('SignatureBox content', () => {
+  it('veteranSignatureContent has not changed', () => {
+    const mockVeteranSignatureContent = [
+      'I certify that I give consent to the individual(s) named in this application to perform personal care services for me upon being approved as Primary and/or Secondary Family Caregivers in the Program of Comprehensive Assistance for Family Caregivers.',
+    ];
+
+    expect(mockVeteranSignatureContent).to.deep.equal(veteranSignatureContent);
```
Comment (mchelen-gov) [inline src/applications/caregivers/tests/components/PreSubmitInfo/index.spec.js:15]: UTs should test program logic, not variable or config values. lmk if i can help

**Review by mchelen-gov — CHANGES_REQUESTED** [review]

File: src/applications/caregivers/tests/components/PreSubmitInfo/index.spec.js (hunk 1 above)
Comment (LinnJS) [reply src/applications/caregivers/tests/components/PreSubmitInfo/index.spec.js:15]: After Mike and I spoke on the phone of why this implementation exist we decided to move this test to Cypress for a more scalable testing solution.

File: src/applications/caregivers/tests/e2e/1010cg.cypress.spec.js [hunk 2]
```diff
@@ -1,15 +1,59 @@
...
+];
+export const mockSecondaryCaregiverContent = [
+  'I certify that I am at least 18 years of age.',
+  "I certify that I am a family member of the Veteran named in this application or I reside with the Veteran, or will do so upon designation as the Veteran's Secondary Family Caregiver.",
+  'I agree to perform personal care services as the Secondary Family Caregiver for the Veteran named on this application.',
+  'I understand that the Veteran or Veteran’s surrogate may request my discharge from the Program of Comprehensive Assistance for Family Caregivers (PCAFC) at any time. I understand that my designation as a Secondary Family Caregiver may be revoked or I may be discharged from the program by the Secretary of Veterans Affairs or his designee, as set forth in 38 CFR 71.45.',
+  'I understand that participation in Program of Comprehensive Assistance for Family Caregivers does not create an employment relationship between me and the Department of Veterans Affairs.',
+];
+
+const checkContent = (partyLabel, content, mockContent) => {
```
Comment (LinnJS) [inline src/applications/caregivers/tests/e2e/1010cg.cypress.spec.js:36]: Finds content by partyLabel then finds every string on content array and makes sure it matches mocked content, match is case sensitive.

**Review by jonathanjnelson — APPROVED** [review]
Looks good @LinnJS

**Review by mchelen-gov — APPROVED** [review]
E2E confirmed to catch changes in both config and code

**Pull request merged**